qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -59,7 +59,7 @@ public void passesCorrectFilesWithNoExceptions() throws Exception {
      * @throws Exception If something wrong happens inside
      */
     @Ignore
-    @Test(expected = ValidationException.class)
+    @org.junit.Test(expected = ValidationException.class)
```

Comment:
@paulodamaso this should be from jupiter

Reply:
@krzyk I've not changed all tags from Jnit 4, as it would take way more than 30 min. I've left those `expected` tags to be fixed later in the `@todo`